Userland/Libraries/LibWeb/CSS/StyleComputer.h

```diff
@@ -102,12 +103,40 @@ struct MatchingRule {
 };
 
 struct FontFaceKey {
-    FlyString family_name;
+    Nonnull<FlyString const> family_name;
```

Comment:
This seems a bit flimsy. We're now forcing clients of this API to be aware that `FontFaceKey` actually doesn't own the `family_name` field, and its lifetime is externally managed.

Isn't there some way we can avoid the ref-count churn without introducing confusing lifetime assumptions in the API?

Reply:
The only use of `FontFaceKey` in the API is by const reference, here:

https://github.com/LadybirdBrowser/ladybird/blob/9dc0fcb1d335e86d89e7b17558926dc980363c61/Userland/Libraries/LibWeb/CSS/StyleComputer.h#L212

We could pass these fields as `const&` parameters directly, though, if passing them through `FontFaceKey const&` is confusing.

Reply:
> We could pass these fields as `const&` parameters directly, though, if passing them through `FontFaceKey const&` is confusing.

Right, unbundling the key into three separate parameters seems like a less confusing way to achieve the same thing :)